.travis.yml

```diff
@@ -1,13 +1,25 @@
 sudo: false # http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+
 language: node_js
+
 node_js:
   - 0.12
   - 4
   - 5
   - 6
 
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm i -g npm@latest
+  - npm install
```

Comment:
What improvement that makes? Also global install doesn't make sense in case of this package as it doesn't expose any binaries